modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +189,33 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+A Mule application can contain several batch job definitions. Each definition 
+has its own scheduling strategy. A *Scheduling Strategy* enables you to control 
+how instances of a given batch job are executed: 
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+state at the same time, the instances execute one at a time based on their
+creation timestamp.
+* ROUND_ROBIN: This setting attempts to execute all available instances of a
+batch job using a round-robin algorithm to assign the available resources. 
++
+This option is useful when you can guarantee that no job execution can have a 
+side effect on another job execution. Therefore, this option is not a good 
+choice for data synchronization jobs, which can update the same record in two 
+concurrent jobs. Because the order of the job execution is not guaranteed, 
+you might end up with a prior version of the data. However, you can safely 
```

Comment:
Because I believe that "end up with" is a bit idiomatic...:
```suggestion
your result might be a prior version of the data. However, you can safely 
```